build.gradle

```
@@ -664,6 +665,24 @@ gitPublish {
 tasks.githubRelease.dependsOn tasks.buildPackageAll
 
 
+/****************
+ * Dockerhub Release *
+ ***************/
+
+jib {
+    from {
+        image = 'openjdk:11-jre-slim-buster'
+    }
+    to {
+        image = 'hivemq/mqtt-cli'
+        tags = [project.version.toString()]
+        auth {
+            username = System.getenv("DOCKER_USER") == null ? "" : System.getenv("DOCKER_USER")
+            password = System.getenv("DOCKER_PASSWORD") == null ? "" : System.getenv("DOCKER_PASSWORD")
```

```suggestion
            username = Objects.requreNonNullOrElse(System.getenv("DOCKER_USER"), "")
            password = Objects.requreNonNullOrElse(System.getenv("DOCKE_PASSWORD"), "")
```
But, idk if this works in groovy :D

---

This is intended, so that you can use `./gradlew jibDockerBuild` to jib a local docker image.